core/test/unit/DatabaseConnector.ts

```diff
@@ -29,6 +29,13 @@ const dbConfig: SeederDatabaseConfig = {
   name: 'database',
 };
 
+const dbConfigv3_6: SeederDatabaseConfig = {
```

Comment:
Please move the config object to test case `should return valid DB connection URI with Mongo 3.6 protocol`:

```
  it('should return valid DB connection URI with Mongo 3.6 protocol', () => {
      const dbConfig: SeederDatabaseConfig = {
            ...
      };
      const expectedUri = 'mongodb+srv://127.0.0.1/database';
      const uri = databaseConnector.getDbConnectionUri(dbConfig);
      expect(uri).toBe(expectedUri);
  })
```

Also, please rename it to simply `dbConfig`.

Reply:
This should be addressed now. Thanks or the feedback!